README.md

```diff
@@ -165,13 +165,28 @@ You find minimal example apps that illustrate the commands at [./examples](./exa
     * ``i-provide-colors``,
     * ``root``.
 
-    ``binge nuke fileName`` removes the specified file from all local packages and the root package, analogous to ``binge copy``.
+    ``binge nuke target`` removes the specified file from all local packages and the root package, analogous to ``binge copy``.
 
-* ``trace targetBranch, ?outputFolder``:
+* ``remove <dependencies...>``
+
+    Removes one or multiple specified dependencies from the current package.
+    For example, running  ``binge remove invariant`` in [./examples/bootstrap/i-filter-stuff](./examples/bootstrap/i-filter-stuff) removes the dependency ``invariant`` from [./examples/bootstrap/i-filter-stuff/package.json](./examples/bootstrap/i-filter-stuff/package.json).
+
+    ``binge remove --all dependencyName`` removes one or multiple specified dependencies from *all* packages.
+
+* ``trace <targetBranch>``:
 
     Compares the current branch with the target branch, transitively finding changed files.
     Outputs the list of affected local packages.
     ``trace`` helps optimizing the test set your CI server runs when checking a merge request by considering the packages that changed *and* the packages that depend on changed packages.
+    If the ``outputDirectory`` argument is provided, binge adds the following three files to the specified path:
+
+    * ``karma.txt``: Lists all packages that have ``testMode`` in the ``.bingerc`` file (see below) set to ``karma``.
+    * ``mocha.txt``: Lists all packages that have ``testMode`` set to ``mocha``-
+    * ``none.txt``: Lists all packages that have ``testMode`` set to ``none`` or no test mode specified.
+
+    The contents of these files provide your CI server with more details on which tests it should run.
+
```

Comment:
The thing here is that, any key that you put there should generate a file. I mean it is not done at the moment, but it is a trivial change. You think this fits the docs?

Reply:
Let's document it when the change is implemented.
But if this is not a _must have_ for version 5, let's do it afterwards as the first issue that then follows a clear workflow: issue created -> issue branch -> tests green -> merged to master -> release tagged & issue linked in change log -> update released.
Just a suggestion, of course :-)